ruby/ql/src/queries/variables/DeadStoreOfLocal.ql

```diff
@@ -6,6 +6,7 @@
  * @problem.severity warning
  * @id rb/useless-assignment-to-local
  * @tags maintainability
+ *       quality
```

Comment:
What queries are supposed to get the `quality` tag?  Should we be adding it to existing and new queries (e.g. `cpp/unused-static-function`, `rust/unused-variable`)?

Reply:
I think eventually all "quality" queries should have the `quality` tag.

Reply:
https://github.com/github/codeql/pull/19455